pymc3/distributions/continuous.py

```diff
@@ -2224,6 +2224,24 @@ def logcdf(value, alpha, beta):
         )
 
 
+class HalfCauchyRV(RandomVariable):
+    name = "halfcauchy"
+    ndim_supp = 0
+    ndims_params = [0]
+    dtype = "floatX"
+    _print_name = ("C**+", "\\operatorname{C^{+}}")
```

Comment:
It seems like you should be able to subclass the existing `HalfCauchyRV`, just like `MultinomialRV` does.  That should prevent the need to duplicate these class-level variables, and preserve our ability to do things like `isinstance(..., HalfCauchyRV)` on instances of this new class.

Reply:
I tried to do that first but I didn't find a way to deal with the `__call__` methods as that invokes `super.__call__`, which I think is strongly linked to the two parameter logic of the original HalfCauchyRV?